setup.py

```diff
@@ -33,15 +28,11 @@ def long_description():
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3.2',
         'Programming Language :: Python :: 3.3',
```

Comment:
Perhaps we can drop these 2 lines as well? Python 3.2 and 3.3 are also deprecated: https://devguide.python.org/#status-of-python-branches

```suggestion
```